handsontable/src/utils/valueAccessors.js

```diff
@@ -0,0 +1,37 @@
+import { isFunction } from '../helpers/function';
+
+/**
+ * Get the value to be set in the cell.
+ *
+ * @param {*} value Initial value.
+ * @param {object} cellMeta The cell meta object.
+ * @returns {*} The value to be set in the cell.
+ */
+export function getValueSetterValue(value, cellMeta) {
+  const { instance, visualRow, visualCol } = cellMeta;
+  const valueSetter = cellMeta.valueSetter ?? null;
+
+  if (isFunction(valueSetter)) {
+    return valueSetter.call(instance, value, visualRow, visualCol);
+  }
+
+  return value;
+}
+
+/**
+ * Get the value to be displayed in the cell.
+ *
+ * @param {*} value Initial value.
+ * @param {object} cellMeta The cell meta object.
+ * @returns {*} The value to be displayed in the cell.
+ */
+export function getValueGetterValue(value, cellMeta) {
+  const { instance } = cellMeta;
+  const valueGetter = cellMeta.valueGetter ?? null;
+
+  if (isFunction(valueGetter)) {
+    return valueGetter.call(instance, value);
+  }
```

Comment:
I think there is a missing piece here about what cell is processed. Otherwise, how would you format a number, date, or something on the fly without knowing the context (`cellMeta`)?
```suggestion
export function getValueGetterValue(value, cellMeta) {
  const { instance, visualRow, visualCol } = cellMeta;
  const valueGetter = cellMeta.valueGetter ?? null;

  if (isFunction(valueGetter)) {
    return valueGetter.call(instance, value, visualRow, visualCol);
  }
```

Another question that comes to my mind is, WDYT about passing the `cellMeta` to both `valueSetter` and `valueGetter` as the last argument? It seems to be helpful for devs who only want coordinates, but also for advanced formatting, the `cellMeta` will be more useful here 🤔